### ISSUE TITLE
Ailments break boss battles with hp thresholds CS3/CS4

### ISSUE THREAD

**Issue by zany130** [AdmiralCurtiss/SenPatcher#109]
Don't know if this would be out of the scope of this project but ... 

inflicting an ailment on bosses where you need to clear a certain threshold of their hp causes the fight to continue past that threshold until the ailment goes away.

 Never really noticed this until the last fight in CS4 intermission where you need to lower both of the boss's hp to their threshold at the same time. This bug makes this impossible to do with a Thor master quartz strategy.
 
 Heard this bug affects CS3/CS4

**Comment by AdmiralCurtiss** [comment]
Huh, fascinating. I guess this never really comes up because bosses usually have high resist and are only affected for 1 turn even if they get hit by a status...

**Comment by zany130** [comment]
Yeah but if you play with soft mod boses have less resistance that's how I encountered the issue while playing the mod at first i thought it might be a bug in the mod but others on soft discord mentioned it happens on vanilla also though its less likely to be an issue there
